src/d3d/nvapi_d3d_low_latency_device.cpp

```diff
@@ -61,4 +114,15 @@ namespace dxvk {
 
         return d3dLowLatencyDevice;
     }
+
+    LowLatencyFrameIdGenerator* NvapiD3dLowLatencyDevice::GetFrameIdGenerator(IUnknown* device) {
+        std::scoped_lock lock(m_lowLatencyFrameIdGeneratorMutex);
+        auto it = m_frameIdGeneratorMap.find(device);
+        if (it != m_frameIdGeneratorMap.end())
+            return it->second.get();
+
+        m_frameIdGeneratorMap.emplace(device, std::make_unique<LowLatencyFrameIdGenerator>());
+
+        return m_frameIdGeneratorMap.find(device)->second.get();
```

Comment:
I believe `emplace` already gives us the iterator we need:

```suggestion
        return m_frameIdGeneratorMap.emplace(device, std::make_unique<LowLatencyFrameIdGenerator>()).first->second.get();
```